frb_codegen/src/commands.rs

```diff
@@ -274,3 +280,91 @@ pub fn build_runner(dart_root: &str) -> Result {
     }
     Ok(())
 }
+
+fn guess_context(dart_root: &str) -> anyhow::Result<Context> {
```

Comment:
uber-nit: Btw what do you think about extracting all these into a new file

Reply:
I personally think it's unnecessary right now since it's very small (especially with method refactoring as you suggested above).